### PR TITLE
Update changelog for v8.0.7

### PR DESCRIPTION
**Pull request by Gudahtt**
Changes that were not user-facing have been omitted, and the descriptions have been updated to be more readable for users.

### CODE REVIEW

**Review by rekmarks — APPROVED** [review]
LGTM!

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,16 @@
...
 
 ## 8.0.7 Fri Aug 07 2020
+- [#9065](https://github.com/MetaMask/metamask-extension/pull/9065): Change title of "Reveal Seed Words" page to "Reveal Seed Phrase"
+- [#8974](https://github.com/MetaMask/metamask-extension/pull/8974): Add tooltip to copy button for contacts and seed phrase
+- [#9063](https://github.com/MetaMask/metamask-extension/pull/9063): Fix broken UI upon failed password validation
+- [#9075](https://github.com/MetaMask/metamask-extension/pull/9075): Fix shifted popup notification when browser is in fullscreen on macOS
+- [#9085](https://github.com/MetaMask/metamask-extension/pull/9085): Support longer text in network dropdown
+- [#8873](https://github.com/MetaMask/metamask-extension/pull/8873): Fix onboarding bug where user can be asked to verify seed phrase twice
+- [#9104](https://github.com/MetaMask/metamask-extension/pull/9104): Replace "Email us" button with "Contact us" button
+- [#9137](https://github.com/MetaMask/metamask-extension/pull/9137): Fix bug with dapps when an `iframe` is garbage collected
```
Comment (rekmarks) [inline CHANGELOG.md:13]: Hm, I think this would've happen whenever a tab to the same site was closed too.

Comment (Gudahtt) [reply CHANGELOG.md:13]: Ah, excellent point! I'll re-word this.

Comment (Gudahtt) [reply CHANGELOG.md]: Updated! The newer description is _OK_ but not fantastic, so I'd welcome any suggestions.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,16 @@
...
 
 ## 8.0.7 Fri Aug 07 2020
+- [#9065](https://github.com/MetaMask/metamask-extension/pull/9065): Change title of "Reveal Seed Words" page to "Reveal Seed Phrase"
+- [#8974](https://github.com/MetaMask/metamask-extension/pull/8974): Add tooltip to copy button for contacts and seed phrase
+- [#9063](https://github.com/MetaMask/metamask-extension/pull/9063): Fix broken UI upon failed password validation
+- [#9075](https://github.com/MetaMask/metamask-extension/pull/9075): Fix shifted popup notification when browser is in fullscreen on macOS
+- [#9085](https://github.com/MetaMask/metamask-extension/pull/9085): Support longer text in network dropdown
+- [#8873](https://github.com/MetaMask/metamask-extension/pull/8873): Fix onboarding bug where user can be asked to verify seed phrase twice
+- [#9104](https://github.com/MetaMask/metamask-extension/pull/9104): Replace "Email us" button with "Contact us" button
+- [#9137](https://github.com/MetaMask/metamask-extension/pull/9137): Fix bug where `accountsChanged` events stop after a dapp connection is closed.
```
Comment (rekmarks) [inline CHANGELOG.md:13]:
```suggestion
- [#9137](https://github.com/MetaMask/metamask-extension/pull/9137): Fix bug where `accountsChanged` events stopped for all open instances of a dapp if one was closed
```

**Review by rekmarks — APPROVED** [review]
I'm fine with it as-is, but left a suggestion.

File: CHANGELOG.md (hunk 2 above)
Comment (Gudahtt) [reply CHANGELOG.md:13]: 🤔 
This wording doesn't seem applicable to iframes though - the graph iframe we saw isn't exactly an "instance of a dapp"

Comment (Gudahtt) [reply CHANGELOG.md:13]: I guess I'll leave it as-is for now

**Pull request merged**